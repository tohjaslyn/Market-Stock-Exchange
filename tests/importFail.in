add trader1 100
import trader1 DEF 0
import trader1 DEF -1
import trader2 ABC 10
exit
