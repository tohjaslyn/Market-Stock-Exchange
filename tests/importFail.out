$ Success.
$ Could not import product into market.
$ Could not import product into market.
$ No such trader in the market.
$ Have a nice day.
